smart-contracts/README.md

```diff
@@ -59,6 +59,10 @@ Run individual tests: `$ truffle test test/<test_name.js>`
 Expected output of the test suite:
 ![peggytestsuite](https://user-images.githubusercontent.com/15370712/58388940-34fb6700-7fda-11e9-9aef-6ae7b2442a55.png)
 
+### Slither
+
+To run [slither]([)](https://github.com/crytic/slither) over all smart contracts you can run `make slither` in this directory. 
+
```

Comment:
Running make slither on my machine gives an error: `source ./venv/bin/activate
make: source: No such file or directory
make: *** [slither] Error 1`

Could you possible provide more documentation on dependencies before running this?